Fastlane/deployment_lanes.rb

```diff
@@ -75,7 +75,7 @@
   # Create a new GitHub release
   last_non_candidate_tag = latest_github_release_tag
   release_title = "#{tag_name} - Beta"
-  release_output = sh("mint run --silent gitbuddy release -l #{last_non_candidate_tag} -b develop --skip-comments --json --use-pre-release --target-commitish #{branch_name} --tag-name #{tag_name} --release-title '#{release_title}'")
+  release_output = sh("mint run --silent gitbuddy release -l #{last_non_candidate_tag} -b #{branch_name} --skip-comments --json --use-pre-release --target-commitish #{branch_name} --tag-name #{tag_name} --release-title '#{release_title}'")
```

Comment:
Hmm, are you sure this is needed? What issue where you running into?
We already configure `--target-commitish #{branch_name}`, for example. I wonder whether that's sufficient

Reply:
When I tried to create a beta build from the main it crashed because we targeted develop. But I'm not sure what this argument does, can you elaborate on that a bit more?

Reply:
See the documentation of GitBuddy:
- [--target-commitish](https://github.com/WeTransfer/GitBuddy/blob/master/Sources/GitBuddyCore/Commands/ReleaseCommand.swift#L36)
- [-b aka --baseBranch](https://github.com/WeTransfer/GitBuddy/blob/master/Sources/GitBuddyCore/Commands/ReleaseCommand.swift#L70)

It's strange that it crashed, since `-b develop` only means it's generating a changelog based on comparing with the `develop` branch. What crash did you run into?

Reply:
Thanks! In that case the change makes sense. When we create a beta build for a hotfix, we want main to be the base branch to compare with for generating the changelog. If develop is the base branch, the changelog will include issues fixed in develop but not in the hotfix.

I think it crashed because when you create a beta build for a hotfix it checks out the main branch instead of develop, thus it can't find it.

Reply:
Gotya, that makes sense! Thanks for getting to the bottom of this with me 🙂

Reply:
Thank you too!